### ISSUE TITLE
Problem: ssh-agent 'bind: address already in use'

### ISSUE THREAD

**Issue by mbusiek-nasuni** [ansible-semaphore/semaphore#2073]
### Issue

Task templates using inventories with the new 'Repository' field fail with the following error: 

````
7:48:05 AM
Failed to install inventory: listening on socket "/tmp/semaphore/ssh-agent-2-1717069685.sock": listen unix /tmp/semaphore/ssh-agent-2-1717069685.sock: bind: address already in use
7:48:05 AM
Running playbook failed: listening on socket "/tmp/semaphore/ssh-agent-2-1717069685.sock": listen unix /tmp/semaphore/ssh-agent-2-1717069685.sock: bind: address already in use
````

I can run the exact same task template using an inventory pointing to the same file, with the only difference being that I did not select the repository field, and left it blank.

### Impact

Ansible (task execution)

### Installation method

Package

### Database

BoltDB

### Browser

Firefox

### Semaphore Version

2.9.109-e32ce6b-1716748050

### Ansible Version

```bash
ansible [core 2.16.6]
```

### Logs & errors

````
May 30 07:48:01 semaphore semaphore[920]: time="2024-05-30T07:48:01-04:00" level=info msg="Task 2147483568 added to queue"
May 30 07:48:05 semaphore semaphore[920]: time="2024-05-30T07:48:05-04:00" level=info msg="Set resource locker with TaskRunner 2147483568"
May 30 07:48:05 semaphore semaphore[920]: time="2024-05-30T07:48:05-04:00" level=info msg="Task 2147483568 removed from queue"
May 30 07:48:05 semaphore semaphore[920]: time="2024-05-30T07:48:05-04:00" level=info msg="Stopped running TaskRunner 2147483568"
May 30 07:48:05 semaphore semaphore[920]: time="2024-05-30T07:48:05-04:00" level=info msg="Release resource locker with TaskRunner 2147483568"
````

### Manual installation - system information

````
Linux semaphore 6.8.0-31-generic #31-Ubuntu SMP PREEMPT_DYNAMIC Sat Apr 20 00:40:06 UTC 2024 x86_64 x86_64 x86_64 GNU/Linux
````

Using NGINX reverse proxy

### Configuration

![image](https://github.com/semaphoreui/semaphore/assets/88546911/65f57432-1929-47db-8c74-b27425ccff59)

![image](https://github.com/semaphoreui/semaphore/assets/88546911/dec9df30-466b-42b8-9c35-8e160ddf5577)

### Additional information

this presumably only affects version >=2.9.109-e32ce6b-1716748050, since that's when the field was introduced

**Comment by fiftin** [comment]
Hi @mschmitt-nasuni thank you for the report. I will need your help to test fix of the bug. Will you help?

**Comment by mbusiek-nasuni** [comment]
@fiftin sure thing, just let me know what you need me to do

**Comment by fiftin** [comment]
Mi @mschmitt-nasuni Please try `v2.9.113-beta`.

**Comment by mbusiek-nasuni** [comment]
@fiftin I don't see a `v2.9.113-beta`, but I see `v2.9.13-beta` was released yesterday. I upgraded to that (even tho dpkg complained that I was downgrading) and the repository field in the Inventories works now! I see it as (optional) and when I run my template with that inventory (with the optional repository defined), I no longer get the bug described here, and the playbook completes successfully.
